doc-Infrastructure_Migration_Solution_Guide/cfme/doc/modules/con_Troubleshooting.adoc

```diff
@@ -0,0 +1,62 @@
+====
+
+. Create and run a new migration plan, using the CSV file to import the virtual machines.
+
+[[Retrying_a_Migration_Plan]]
+[discrete]
+== Retrying a Migration Plan
+
+If one or more virtual machines in a migration plan fails to migrate, the plan's migration status is `Failed`. You can retry a migration plan to migrate the virtual machines that failed to migrate:
+
+. Click the migration plan to go to the details page to identify the virtual machines that failed to migrate.
+. Delete the target virtual machines corresponding to the source virtual machines that failed to migrate. Otherwise, the retry will fail because a target machine with the same name already exists.
+. Go to menu:Compute[Migration], select `Failed Migrations`, select the failed migration plan, and click `Retry`.
+
+[discrete]
+== Log Files
+
+If a migration plan does not complete successfully, you can check the logs for errors:
+
+* CloudForms migration log: `/var/www/miq/vmdb/log/automation.log`
+* Conversion host logs: `/var/log/vdsm/import/`
+* Virtual machine migration log: Go to menu:Compute[Migrate], click the migration plan to go to the details page, and click `Download Log`.
++
+.Migrated Virtual Machine Logs
+image:Migrated_VM_details.png[]
+
+[discrete]
+== Common Issues and Mistakes
+
+Incorrect infrastructure mapping::
+If you create an infrastructure mapping and subsequently add or remove providers or provider objects, the infrastructure mapping will not work because the object IDs of the providers and their objects have changed. You must delete the infrastructure mapping and re-create it.
+
+RHV host credentials not set in CloudForms::
+If the RHV host credentials are not set, CloudForms cannot communicate with the hosts. To authenticate the RHV hosts with CloudForms, see link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6/html-single/managing_providers/#authenticating_rhv_hosts[Authenticating Red Hat Virtualization Hosts] in _CloudForms: Managing Providers_.
+
+Incorrect attributes in the virtual machine CSV import file::
+Create a new migration plan, import the correct CSV file, and run the migration plan.
+
+Migration fails immediately::
+If a migration fails immediately, before the progress bar is displayed, this is normally caused by a configuration error in the environment. Check the *automation.log* file on the CloudForms appliance.
+
+Migration fails at virtual machine conversion::
+If a migration fails when the virtual machines are being converted (the progress bar updates), click the migration plan to go to the details page and download the conversion log for the the virtual machine(s) that failed to migrate.
+
+SSH transformation fails::
+If you are using SSSD with single-sign-on for RHV, SSH fails for the vdsm account. See  link:https://bugzilla.redhat.com/show_bug.cgi?id=1544379[] for details.
```

Comment:
"single-sign-on" -> "single sign-on" 

Backticks around vdsm

Reply:
Thanks. The first was a copy/paste error. Missed it.
Done